stories/editorialMarking/bylineImage.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import {
+	StoryTitle,
+	StorySubtitle,
+	DemoContainer,
+	SiteSelection,
+} from '../storybook-components';
+
+import LeetTextarea from '../../src/atoms/LeetTextarea';
+
+import Image from '../../src/atoms/Image';
+import BylineImage from '../../src/atoms/BylineImage';
+
+export default () => (
+	<SiteSelection>
```

Comment:
The SiteSelection is included [here](https://github.com/dbmedialab/shiny/pull/30/files#diff-156d33391f29ea7da619ba648e042ea0R14) already, this one can be removed

Reply:
Removed.